src/plugins/ClausePlugin.js

```diff
@@ -59,40 +58,6 @@ function ClausePlugin() {
     return newSchema;
   });
 
-  /**
-   * Called by the clause plugin into the contract editor
-   * when we need to load a template
-   */
-  async function loadTemplate(templateUri) {
-    let template = templates[templateUri];
-    if (!template) {
-      console.log(`Loading template: ${templateUri}`);
-      template = await Template.fromUrl(templateUri);
-      templates[templateUri] = template;
```

Comment:
I'm unclear how the load template will work once it is removed. A function `loadTemplate` outside the clause plugin will not have access to the `templates` variables.

Reply:
`templates` isn't used. It's just a local variable. This logic will be passed down from the above app and used below as `loadTemplateCallback`